Formula/envconsul.rb

```diff
@@ -0,0 +1,28 @@
+class Envconsul < Formula
+  desc "Launch process with environment variables from Consul and Vault"
+  homepage "https://github.com/hashicorp/envconsul"
+  url "https://github.com/hashicorp/envconsul/archive/v0.7.2.tar.gz"
+  sha256 "c2b2723089f82f7b1623676fda8d378795bf87b4dbc6d4b297e5fc27aeab0aca"
+  depends_on "go" => :build
+
+  def install
+    gopath = buildpath/"go"
```

Comment:
Please take a look at some of the other formulae that depend on `go` to make this install method more idiomatic. For example, see `flint-checker` and `goose`.

Reply:
Done, made it much easier indeed. Thanks!